images/base/Dockerfile

```diff
@@ -12,51 +12,53 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# kind cluster base image
+# kind node base image
 #
 # For systemd + docker configuration used below, see the following references:
 # https://www.freedesktop.org/wiki/Software/systemd/ContainerInterface/
-# https://developers.redhat.com/blog/2014/05/05/running-systemd-within-docker-container/
-# https://developers.redhat.com/blog/2016/09/13/running-systemd-in-a-non-privileged-container/
 
-ARG BASE_IMAGE="ubuntu:19.10"
-FROM ${BASE_IMAGE}
+# start from ubuntu 19.10, this image is reasonably small as a starting point
+# for a kubernetes node image, it doesn't contain much we don't need
+FROM ubuntu:19.10
 
-# setting DEBIAN_FRONTEND=noninteractive stops some apt warnings, this is not 
-# a real argument, we're (ab)using ARG to get a temporary ENV again.
-ARG DEBIAN_FRONTEND=noninteractive
+# Configure containerd and runc binaries from kind-ci/containerd-nightlies repository
+# The repository contains latest stable releases and nightlies built for multiple architectures
+ARG CONTAINERD_VERSION="v1.3.0-7-g0b43a311"
+# Configure CNI binaries from upstream
+ARG CNI_VERSION="v0.8.2"
+# Configure crictl binary from upstream
+ARG CRICTL_VERSION="v1.16.1"
 
-COPY clean-install /usr/local/bin/clean-install
-RUN chmod +x /usr/local/bin/clean-install
+# copy in static files (configs, scripts)
+COPY files/ /
```

Comment:
This is a bit risky but simpler

Reply:
what is risky about it? 🙃

Reply:
drop or delete a file that gets into the image accidentally

Reply:
"drop or delete" ?
how are files getting into the image accidentally?

Reply:
I mean, if you add or remove one file in that folder `files/` accidentally, it will be included in the image and probably you won't notice. 
It can happen, it's difficult that something like that happens, but is it possible since we are adding the folder recursively and copying it to the root folder

Reply:
it should not be difficult to notice. We should be reviewing changes to this directory regardless.

Reply:
adding or removing will show up in git